README.md

```diff
@@ -0,0 +1,345 @@
+
+      switch(op)
+      {
+          case "+":
+              return Response.ok(a + "+" + b + "=" + (Integer.toString((numA + numB)))).build();
+
+          case "-":
+              return Response.ok(a + "-" + b + "=" + (Integer.toString((numA - numB)))).build();
+
+          case "*":
+              return Response.ok(a + "*" + b + "=" + (Integer.toString((numA * numB)))).build();
+
+          default:
+              return Response.ok("Invalid operation. Please Try again").build();
+      }
+    }
+}
+```
+Any changes you make to your code is automatically built and re-deployed by Codewind and viewed in your browser. 
+
+### Working with the microservice
+
+You now can work with your calculator.
+
+1. Use the port number you saw when you first opened the application.
+2. Make sure to remove the `< >` symbol in the URL. 
+3. `http://127.0.0.1:<port>/starter/calculator/aboutme` 
+4. You should see the following response:
+
+```
+You can add (+), subtract (-), and multiply (*) with this simple calculator.
+```
+
+You could also try a few of the sample calculator functions: 
+
+* `http://127.0.0.1:<port>/starter/calculator/{op}/{a}/{b}`, where you can input one of the available operations `(+, _, *)`, and an integer a, and an integer b.
+* So for `http://127.0.0.1:<port>/starter/calculator/+/10/3` you should see: `10+3=13`.
+
+### More Learning
+
+You have created a simple microservice using the VS Code IDE. For further learning:
+
+* Try [additional Kabanero guides](https://www.kabanero.io/guides) available for each Collection: Eclipse MicroProfile, Springboot, Node.js.
+* Learn more about [Codewind](https://www.eclipse.org/codewind/).
+* Review [project commands for Codewind for VS Code](https://www.eclipse.org/codewind/mdt-vsc-commands-project.html).
+
+## Developing with Eclipse 
+
+If you use Eclipse, you can use Codewind for Eclipse to develop and debug your containerized projects from within a local Eclipse IDE.
+
+### Prerequisite
+
+Before you can develop a microservice with Eclipse, you need to:
+
+* [Install Docker](https://docs.docker.com/install/) 
+    * **Note:** Make sure to install or upgrade to minimum Docker version 19.03. 
+* [Install Eclipse](https://www.eclipse.org/downloads/packages/release/)
+    * **Note:** Make sure to install or upgrade to mimimum Eclipse version 2019-09 R (4.13.0). 
+
+### Installing Codewind for Eclipse
+
+The Codewind installation includes two parts:
+
+1. The Eclipse plug-in installs when you install Codewind from the [Eclipse Marketplace](https://marketplace.eclipse.org/content/codewind)or when you install by searching in the `Eclipse Extensions` view.
+2. The Codewind backend containers install after you click `Install`. After you click `Install`, the necessary images are downloaded from the internet. The download is approximately 1 GB.
+
+The following images are pulled. These images together form the Codewind backend:
+
+1. `eclipse/codewind-performance-amd64`
+2. `eclipse/codewind-pfe-amd64`
```

Comment:
If you make changes suggested above, suggest you also change this section as well.